### DISCUSSION TITLE
JellyChain1 User Interface

### DISCUSSION THREAD

**Discussion by Cotopia-Inc** [JellyChain1/JellyChain1#8] [Announcements]
We are currently in the process of building a new UI. If anyone has any input they want to say or change or add this is an open discussion for the subject at hand. This is your time to get your opinion out. 
We will open a repository for the UI soon and the general public is welcome to contribute or collaborate however you feel. We look forward to a great UI.

**Comment by BotGame22** [comment]
You have my full support.

**Comment by HeartBreakKid8** [comment]
I hope it be simple and easy to read.  because I went to one site and it was really hard to navigate around and read what was going on.
